src/domains/transaction/hooks/use-profile-transactions.ts

```diff
@@ -85,7 +87,7 @@ export const useProfileTransactions = ({
 
 			setState((state) => ({
 				...state,
-				hasMore: items.length > 0 && response.hasMorePages(),
+				hasMore: items.length >= LIMIT && response.hasMorePages(),
```

Comment:
@w3ea overall looking good :+1: 
One minor,  I think we need to check `response.hasMorePages()` only if it the items >= LIMIT. Otherwise I think the `hasMorePages()` check should be ignored

```suggestion
				hasMore: items.length >= LIMIT ?? response.hasMorePages(),
```

Reply:
@goga-m we can't just check `response.hasMorePages()`
[these transactions](https://github.com/PayvoHQ/wallet/blob/develop/src/domains/transaction/hooks/use-profile-transactions.ts#L84) are relevant to all wallets
imagine we have 7 wallets with 5 transactions
7*5 > limit (30)

Reply:
plus, i think it does [the same thing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND#:~:text=If%20expr1%20can%20be%20converted%20to%20true%2C%20returns%20expr2%3B%20else%2C%20returns%20expr1.) now

Reply:
@w3ea cool :+1:
Yeah it's completely fine as it is, not a major issue. I think we are covering all the cases here. My suggestion  was more to make it more readable, something like:
```js
const hasMorePages = () => {
    if(items.length < LIMIT) {
        return false;
    }
    return response.hasMorePages()
}
```

Reply:
@goga-m ah you meant readability :)
what do you think?
https://github.com/PayvoHQ/wallet/pull/3/commits/6bfdce1fbbbe41d83a3ab2b5307051843edca5ff

Reply:
Sweet!

Reply:
:catjam: